python-api/integrations/openrouter/client.py

```
@@ -0,0 +1,301 @@
+                ) as resp:
+                    self._raise_for_status(resp)
+                    async for line in resp.aiter_lines():
+                        line = line.strip()
+                        if not line or not line.startswith("data:"):
+                            continue
+                        data = line[len("data:"):].strip()
+                        if data == "[DONE]":
+                            break
+                        try:
+                            yield ChatCompletionChunk.model_validate(json.loads(data))
+                        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)